packages/react-components/src/components/ProductSwitcher/ProductSwitcher.mdx

```diff
@@ -0,0 +1,84 @@
+import { Meta, Title, Canvas, ArgTypes } from '@storybook/blocks';
+
+import * as ProductSwitcher from './ProductSwitcher.stories';
+
+<Meta of={ProductSwitcher} />
+
+<Title>ProductSwitcher</Title>
+
+[Intro](#Intro) | [Component API](#ComponentAPI) | [Content Spec](#ContentSpec)
+
+## Intro <a id="Intro" />
+
+`ProductSwitcher` is a component used to switch between products in the LiveChat ecosystem. It allows users to switch between products they have access to, see their subscription status, and navigate to the product dashboard.
+
+<Canvas of={ProductSwitcher.Default} sourceState="none" />
+
+#### Example implementation
+
+ProductSwitcher is a controlled component. To use it, you need to provide the `mainProductId` (type ProductId) and `productOptions` (type ProductOption) props.<br/>
+
+- The `mainProductId` prop is required and should be the id of the product that the user is currently using.<br/>
+  The available options are: `livechat`, `helpdesk`, `chatbot`, `knowledgebase`, `openwidget`, `hello`, `accounts`. They are described in the TS type value called `ProductId`.<br/>
+- The `productOptions` prop is an array of products that the user has access to. Object structure is described in type `ProductOption`.<br/>
+  The `productOptions` can be generated using the `useProductSwitcher` hook. The hook requires the following props:<br/>
+  - `env` - the environment in which the product switcher is used. It can be either `labs`, `staging` or `prod` .<br/>
+  - `installedProducts` - an array of objects with the `product` key. The `product` key should be one of the available products: `livechat`, `helpdesk`, `chatbot`, `knowledgebase`, `openwidget`, `hello`, `accounts`.<br/>
```

Comment:
`platform` will be added later?

Reply:
exactly, there will be no platform in the switcher for now